fractal/cmd/__init__.py

```diff
@@ -121,6 +122,8 @@ async def task(
         iface = await task_collect_pip(client, batch=batch, **kwargs)
     elif subcmd == "check-collection":
         iface = await task_collection_check(client, **kwargs)
+    elif subcmd == "new":
```

Comment:
This command should handle both `batch=True/False`, because for `fractal XXX new` it is useful to also run `fractal --batch XXX new` that only prints the ID of the newly-created object.